### DISCUSSION TITLE
Change AWS region

### DISCUSSION THREAD

**Discussion by agserrano3** [supabase/supabase#15701] [Questions] [answered]
Hi there,
We have our Supabase instances in us-west-1 and I am trying to change them to us-east-1, but I don't seem to be able to modify that from the interface. Am I missing permissions or is not possible?
Thanks

**Comment by GaryAustin1** [comment, marked as answer]
If it is possible you would need to contact support, or move them using migrations. https://supabase.com/docs/guides/platform/migrating-and-upgrading-projects#migrate-your-project
